### PR TITLE
doc: Update old blogs

### PR DESCRIPTION
**Pull request by shimonp21**

### CODE REVIEW

File: website/pages/blog/introducing-cloudquery-sdk.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { BlogHeader } from "../../components/BlogHeader"
 
 <BlogHeader/>
 
-Today we are pleased to announce the release of [CloudQuery SDK](https://github.com/cloudquery/cq-provider-sdk)!
+Today we are pleased to announce the release of [CloudQuery SDK](https://github.com/cloudquery/plugin-sdk)!
```
Comment (bbernays) [inline website/pages/blog/introducing-cloudquery-sdk.md:13]: I think we should either delete this blog and write a new one or mark it as deprecated...

File: website/pages/blog/announcing-policies-in-cloudquery-hub.md [hunk 2]
```diff
@@ -10,6 +10,8 @@ import { BlogHeader } from "../../components/BlogHeader"
 
 <BlogHeader/>
 
+**HCL policies were deprecated - see up-to-date policy documentation [here](https://www.cloudquery.io/docs/core-concepts/policies)**
```
Comment (hermanschaaf) [inline website/pages/blog/announcing-policies-in-cloudquery-hub.md:13]: Can we use a Callout block here instead?

```
import { Callout } from 'nextra-theme-docs'
```

```
<Callout> 

HCL policies were deprecated - see up-to-date policy documentation [here](https://www.cloudquery.io/docs/core-concepts/policies)

</Callout>
```

Comment (shimonp21) [reply website/pages/blog/announcing-policies-in-cloudquery-hub.md:13]: Hmmm, not sure if our blogs support `.mdx` (they're all `.md` right now).
I think making sure they support it is out-of-scope of this PR?

Comment (yevgenypats) [reply website/pages/blog/announcing-policies-in-cloudquery-hub.md:13]: They support `mdx` and there is even no need to rename it so it should just work

Comment (shimonp21) [reply website/pages/blog/announcing-policies-in-cloudquery-hub.md]: :) !

File: website/pages/blog/introducing-the-github-cloudquery-provider.md [hunk 3]
```diff
@@ -21,63 +21,42 @@ In this tutorial, we will install CloudQuery and use it to fetch GitHub resource
...
-```bash
-docker run -p 5432:5432 -e POSTGRES_PASSWORD=pass -d postgres
-```
-
-Finally, we can execute CloudQuery to fetch all our GitHub resources. Simply run the fetch command and let the magic happen.
-
-```bash
-cloudquery fetch
-```
+- [Download CloudQuery](https://www.cloudquery.io/docs/quickstart/macOS#download-cloudquery-cli).
```
Comment (hermanschaaf) [inline website/pages/blog/introducing-the-github-cloudquery-provider.md:24]:
```suggestion
- [Download CloudQuery](https://www.cloudquery.io/docs/quickstart).
```

Although we lose the ability to link to a section by doing this, I think it's better to redirect to the quickstart page appropriate for the OS you're on.

File: website/pages/blog/introducing-the-github-cloudquery-provider.md [hunk 4]
```diff
@@ -21,63 +21,42 @@ In this tutorial, we will install CloudQuery and use it to fetch GitHub resource
...
+- Acquire a github (personal access token)[https://docs.github.com/en/enterprise-server@3.4/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token]
```
Comment (hermanschaaf) [inline website/pages/blog/introducing-the-github-cloudquery-provider.md:25]:
```suggestion
- Acquire a GitHub (personal access token)[https://docs.github.com/en/enterprise-server@3.4/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token]
```

**Review by hermanschaaf — APPROVED** [review]
I think we should generally favor callouts to notify users when things have changed, rather than trying to keep all blog posts continually up-to-date, but this is good as a once-off for the v0 to v1 transition 👍  

I left some minor suggestions

**Review by yevgenypats — APPROVED** [review]

**Pull request merged**